src/main/java/fi/metatavu/famifarm/rest/V1RESTService.java

```diff
@@ -936,6 +1038,39 @@ public Response updateCampaign(@Valid Campaign campaign, UUID campaignId) {
     return createOk(campaignTranslator.translate(updatedCampaign));
   }
 
+  @Override
+  @RolesAllowed({ Roles.ADMIN, Roles.MANAGER, Roles.WORKER })
+  public Response updateCutPacking(@Valid CutPacking cutPacking, UUID cutPackingId) {
+    fi.metatavu.famifarm.persistence.model.CutPacking existingCutPacking = cutPackingController.find(cutPackingId);
+
+    if (existingCutPacking == null) {
+      return createNotFound("Cut packing with id " + cutPackingId + " not found!");
```

Comment:
Use formatter instead of concat i.e. String.format("Product with id %s not found!", productId)